tests_common/setup.py

```diff
@@ -27,6 +27,7 @@
     "pytest-asyncio",
     "pytest-env",
     "pytest-postgresql",
+    "pyscopg2",
```

Comment:
This is a typo I guess?

Reply:
this is a cherry pick, I did not type anything